.github/workflows/ci.yml

```diff
@@ -16,6 +16,7 @@ jobs:
           - 16
           - 18
           - 19
```

Comment:
Kept Node 19 in here since it doesn't EOL until June: https://nodejs.dev/en/about/releases/